### ISSUE TITLE
Feedback user experience

### ISSUE THREAD

**Issue by Bukanradenz9** [beyond-btc/community#86]
The UI already looks good, just need additional features like mini charts to see price fluctuations and hopefully in the next update there will be additional language features. for performance there should probably be a speed up feature to increase network speed or a feature for dividing transaction batches to be more efficient

**Comment by sunnybeyond** [comment]
Low effort, but the transaction batches feature might be neat?

**Comment by Bukanradenz9** [comment]
yeah, not many dex use that feature yet, maybe you can be one of the pioneers

**Comment by sunnybeyond** [comment]
**Congratulations!** Your review has passed our quality check, and you are now eligible for a The Orange Pills whitelist! To register your address, please follow these steps:

1. Join the Beyond Discord server. (https://join.beyond.tech/) 
2. Open a support ticket in the #support channel.
3. Add a comment to your GitHub entry with the ticket number from your Discord support ticket.
4. In the support ticket, make sure to include:
- Your Taproot address (bc1p)
- A link to your GitHub entry

**Important:**

- Entries missing any of the required information above will NOT be considered and will be closed automatically.
- All entries must be completed by Sunday, November 3rd, 23:59 UTC. Late entries will NOT be accepted.

**Comment by Bukanradenz9** [comment]
Ticket number #628-denny99
Thanks :)
